### ISSUE TITLE
MongoDB

### ISSUE THREAD

**Issue by Aardenfell** [ZiraDiscord/Zira#3]
I've launched my own instance of zira but it is unresponsive to any commands. I think this might be because I've setup MongoDB incorrectly. I'm really unsure of what to enter in the "Host/Collection" slot.
This error is thrown alot too
`[Cluster (Error)] TypeError: Cannot read property 'collection' of null`

**Comment by cbackas** [comment]
Yeah it has something to do with the initial connection to the database, which it apparently only tries once. And then after that it just has the null error you provided. 

Here's the erorr for the initial connection
[ ERROR ] 7/9/2018 15:3:30 [Cluster (Error)] { MongoError: failed to connect to server [ziradb:27017] on first connect [MongoError: getaddrinfo EAI_AGAIN ziradb:27017]

I haven't been able to figure out what to put in the host/collection slot either. I tried just putting "ziradb" hoping it would create said database/collection automatically but that didn't work. I used mongo shell to do "use ziradb" and then created a collection called "zcollection" inside of ziradb which DOES show when I use "show collections" in mongo shell... So I know the collection exists. I then tried just putting "zcollection" in the host/collections field, but that didn't work. I then tried "ziradb/zcollection" and that didn't work either, and just "ziradb" alone still doesn't work. Wish there was a little bit more documentation in the pm2.example.json but this whole shut down/self host thing all happened pretty fast. 

I really just want to get this thing running for a week or so until I can just rewrite its tasks into another bot since this thing won't be maintained anymore..... Does anyone have any clues as to what to put in this host/collection field?

**Comment by FireGamer3** [comment]
Host is the host of the database if you are running it on the same machine as zira it should be localhost  collection is just the Database name

Example 
Localhost/Zira
